src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/kafka/RecordHandler.java

```diff
@@ -1412,6 +1418,38 @@ private void doGetExpectedDefaultMetersRequest(CommandMessage message) {
         }
     }
 
+    private void doDumpGroupsRequest(final CommandMessage message) {
```

Comment:
Can you stop using this approach for "new" code? I suppose to use SpeakerCommand API for "new" code. 

The worst part of this approach - this request will block one of FL thread up to 10 seconds.Not worse but still bad - it uses not reliable "channel" in communication to the switch.